testing_utils/delphi_utils/requirements.txt

```diff
@@ -0,0 +1,7 @@
+covidcast
+delphi-epidata
+epiweeks
+numpy
+pandas==1.5.3
```

Comment:
I think I specifically pinned to the latest available pandas version for 3.8 at the time. It still hasn't changed since then so when I removed the explicit pin it still downloaded 1.5.3

Reply:
DOH, i wasnt thinking!  Of course this requirements list ends up with `pandas==1.5.3` if you remove the explicit version number spec -- it includes `covidcast` which does have 1.5.3 pinned.  [According to the documentation, python3.8 is supported by pandas v2](https://pandas.pydata.org/pandas-docs/version/2.0/getting_started/install.html#python-version-support), so we really should pin 1.5.3 in the delphi_utils package.